### DISCUSSION TITLE
Rk3328的L1Pro 如何安裝最新內核kernel？

### DISCUSSION THREAD

**Discussion by sillydanny** [ophub/amlogic-s9xxx-armbian#1505] [Rockchip]
L1Pro是刷入Flippy大佬以前發怖的armbian (debian buster ) 版本，已經手動更新到bookworm, 但kernel 還是5.15.，請問ophub 大大，可以提供手動刷入最新kernel 的方法嗎？十分謝謝

**Comment by ophub** [comment]
你下系统的度盘里有个升级内核的脚本

**Reply by sillydanny** [reply]
請問大大有度盤的連結嗎？

**Comment by sillydanny** [comment]
完成手動更新，透過大佬ophub 的kernel 文件替換相對應的型號，完成更新內核，觀望一段時間是否穩定。謝謝大大 分享。

**Comment by ophub** [comment]
https://github.com/ophub/kernel/releases/tag/kernel_flippy

这个里面的是flippy分享的，我转存的。其他几个tags里面也是用他的kernel源码编译的，config选项有调整。都通用，可以互换使用。

https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/build-armbian/armbian-files/common-files/usr/sbin/armbian-kernel

这是我的内核更新脚本，你对比f大的3个内核的路径，结构确认一下更新一下就可以使用了。可能就是缺一个/etc/ophub-release文件，你可以下载一下仓库里最新的L1pro系统，把这个文件上传到f大里。

**Reply by sillydanny** [reply]
十分感謝大大分享👍
